delegation.ipldsch

```diff
@@ -0,0 +1,28 @@
+type Delegation struct {
+  dgn &DelegationPayload
+  sig &Signature
+}
```

Comment:
> I would suggest renaming dng field to something more generic like payload, or whatever acronym makes sense so it could be use universally.

💯 yeah I want to do this as part of the (batch) signature spec. This field is kind of pulling double duty as a not-quite-capsule and payload identifier.

Will fix 👍